packages/clay-data-provider/src/useCache.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import LRU from 'lru-cache';
+import stringify from 'fast-json-stable-stringify';
```

Comment:
Whats the reason for bringing in this dependency vs JSON.stringify?

Reply:
This was a problem pointed out by Greg, because we used to validate the cache, as we are using the `variables` to check if the cache satisfies, the order of the object can influence this check, so this lib helps with that.

Reply:
ah gotcha, we actually had that same issue on Faro awhile back and did the same thing. Had forgotten about it till now.